dotcom-rendering/src/web/components/Card/components/CardLink.tsx

```diff
@@ -34,8 +30,11 @@ export const CardLink = ({
 	dataLinkName = 'article',
 }: Props) => {
 	return (
-		<a href={linkTo} css={fauxLinkStyles} data-link-name={dataLinkName}>
-			{headlineText}
-		</a>
+		<a
+			href={linkTo}
+			css={fauxLinkStyles}
+			data-link-name={dataLinkName}
+			aria-label={headlineText}
```

Comment:
👍 this is easier to understand when looking at the code

Reply:
Nicer for when you inspect the DOM too!